datadog/fwprovider/resource_datadog_integration_gcp_sts.go

```diff
@@ -240,6 +249,12 @@ func (r *integrationGcpStsResource) buildIntegrationGcpStsRequestBody(ctx contex
 	diags := diag.Diagnostics{}
 	attributes := datadogV2.NewGCPSTSServiceAccountAttributesWithDefaults()
 
+	accountTags := make([]string, 0)
```

Comment:
This is not idiomatic Go. Typically, you initialize slices to `nil`. `nil` slices are valid for just about anything (i.e. iteration, appending, etc).
`var accountTags []string`